release-docs/0.11/CHANGES.0.11.md

```diff
@@ -0,0 +1,163 @@
+| [FALCON-2293](https://issues.apache.org/jira/browse/FALCON-2293) | falcon extension fails for deletion if 2 extensions have same entity name | extensions | Major | sandeep samudrala | sandeep samudrala |
+| [FALCON-2286](https://issues.apache.org/jira/browse/FALCON-2286) | Falcon upgradation fails to create new tables | extensions | Major | Pracheer Agarwal | Pragya Mittal |
+| [FALCON-2292](https://issues.apache.org/jira/browse/FALCON-2292) | User Extension submission is failing because of the validate late inputs | extensions | Major | sandeep samudrala | sandeep samudrala |
+| [FALCON-2291](https://issues.apache.org/jira/browse/FALCON-2291) | Incorrect path to the REST calls on extension (POST) | docs | Major | Pengfei Xuan | Pengfei Xuan |
+| [FALCON-2290](https://issues.apache.org/jira/browse/FALCON-2290) | Change staging directory for entity creation while extension job submission | extensions | Major | Pracheer Agarwal | Pracheer Agarwal |
+| [FALCON-2287](https://issues.apache.org/jira/browse/FALCON-2287) | Extension list fails with MIME type error on server | extensions | Major | Pracheer Agarwal | Pragya Mittal | | [FALCON-2283](https://issues.apache.org/jira/browse/FALCON-2283) | Internal Server Error : On trying to submit a trusted extension job 
+ | extensions | Major | sandeep samudrala | Richie Varghese | | [FALCON-2279](https://issues.apache.org/jira/browse/FALCON-2279) | Killed instances are not rerun via falcon | rerun | Major | sandeep samudrala | Pragya Mittal |
+| [FALCON-2277](https://issues.apache.org/jira/browse/FALCON-2277) | Incorrect error message on registering trusted exceptions | extensions | Major | Pracheer Agarwal | Richie Varghese |
+| [FALCON-2276](https://issues.apache.org/jira/browse/FALCON-2276) | FalconCLIException while submitting trusted extension job | extensions | Major | sandeep samudrala | Richie Varghese |
+| [FALCON-2274](https://issues.apache.org/jira/browse/FALCON-2274) | Job list in extension is failing with IllegalArgumentException | extensions | Major | Pracheer Agarwal | Pragya Mittal |
+| [FALCON-2273](https://issues.apache.org/jira/browse/FALCON-2273) | Disallow external entity injection and clean up some log messages | general | Major | Venkat Ranganathan | Venkat Ranganathan |
+| [FALCON-2275](https://issues.apache.org/jira/browse/FALCON-2275) | Unable to list instances of an extension | extensions | Major | Pracheer Agarwal | Pragya Mittal |
+
+
+### DOCUMENTATION:
+| JIRA | Summary | Priority | Contributor | Reporter |
+|:---- |:---- |:---- |:---- |:---- |
+| [FALCON-2143](https://issues.apache.org/jira/browse/FALCON-2143) | Documentation of BacklogEmitter Service | Major | Praveen Adlakha | Praveen Adlakha |
+| [FALCON-2078](https://issues.apache.org/jira/browse/FALCON-2078) | Documentation for Process SLA | Major | Praveen Adlakha | Praveen Adlakha |
```

Comment:
Add FALCON-2196 ( User extensions documentation ) to the change list

Reply:
Good catch. That JIRA was still open although PR merged, hence the tool didn't pick it up. Added it.